### PR TITLE
Fixed ControlState Fixed PayFineUI, Fixed Member, Fixed UIState, Fixe…

### PR DESCRIPTION
**Pull request by Nylanderjonathan**
@nhuthuynh Review

### CODE REVIEW

File: src/PayFineControl.java [hunk 1]
```diff
@@ -1,62 +1,61 @@
...
-	private enum CONTROL_STATE { INITIALISED, READY, PAYING, COMPLETED, CANCELLED };
-	private CONTROL_STATE state;
-	
-	private library library;
-	private member member;;
+	private PayFineUI payFineUI;
+	private enum ControlState { INITIALISED, READY, PAYING, COMPLETED, CANCELLED };
+	private ControlState state;
+	private Library library;
+	private Member member;;
```
Comment (nhuthuynh) [inline src/PayFineControl.java]: remove redundant semicolon

File: src/PayFineControl.java [hunk 2]
```diff
@@ -1,62 +1,61 @@
...
-		ui.setState(PayFineUI.UI_STATE.READY);
-		state = CONTROL_STATE.READY;		
+		this.payFineUI = payFineUI;
+		payFineUI.setState(PayFineUI.UIState.READY);
+		state = ControlState.READY;		
 	}
 
-
+        //Checking Card with memberId
 	public void cardSwiped(int memberId) {
```
Comment (nhuthuynh) [inline src/PayFineControl.java]: method name should be verb or verb phrase.

**Review by nhuthuynh — CHANGES_REQUESTED** [review]
Please change as i commented

**Comment by Nylanderjonathan** [comment]
Thanks Minh, fixed.

**Review by nhuthuynh — APPROVED** [review]

**Comment by nhuthuynh** [comment]
@mendefrost @nsingl02 please get the latest code from master

**Pull request merged**